pipestat/parsed_schema.py

```diff
@@ -203,10 +203,21 @@ def build_sample_model(self):
         field_defs = self._add_record_identifier_field(field_defs)
         field_defs = self._add_id_field(field_defs)
         field_defs = self._add_namespace_field(field_defs)
+        field_defs = self._add_project_name_field(field_defs)
         return _create_model(self.sample_table_name, **field_defs)
 
     @staticmethod
     def _add_namespace_field(field_defs: Dict[str, Any]) -> Dict[str, Any]:
+        # TODO Deprecate use of namespace
+        id_key = "project_name"
```

Comment:
probably best to make this a global CONSTANT

Reply:
Done